selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -395,17 +395,13 @@ private String getDefaultVersion(Set<String> keys, String appName) {
     return listOfApps.size() > 0 ? listOfApps.last() : null;
   }
 
-  // throws exception if user didn't pass the capability as a boolean
+  // throws exception if user didn't pass the capability as a boolean or String parsable as boolean
   private Boolean getBooleanCapability(String key) {
     Object o = getRawCapabilities().get(key);
     if (o == null) {
       return null;
-    } else if (o instanceof Boolean) {
-      return (Boolean) o;
     } else {
-      throw new ClassCastException(String.format(
-          "DesiredCapability %s's value should be boolean: found value %s of type %s", key, o.toString(), o.getClass()
-              .getName()));
+      return is(key);
```

Comment:
No exception would be thrown, the value would be 'false'. Is that acceptable/desirable?

From `Boolean.class`
```java
    private static boolean toBoolean(String name) {
        return ((name != null) && name.equalsIgnoreCase("true"));
    }
```